application/src/serializer.cpp

```diff
@@ -41,6 +42,15 @@ QList<Message> Serializer::deserializeToMessageList(const QByteArray &jsonData)
 
         messageList.append(message);
     }
-
     return messageList;
 }
+
+User Serializer::deserializeToUser(const QByteArray &jsonData)
+{
+    QJsonDocument jsonDoc = QJsonDocument::fromJson(jsonData);
+    QJsonObject jsonObj = jsonDoc.object();
```

Comment:
```suggestion
    QJsonObject jsonObj = QJsonDocument::fromJson(jsonData).object();
```
можно сразу так сделать, jsonDoc нигде не используется